### ISSUE TITLE
Installation on Windows

### ISSUE THREAD

**Issue by anufrievroman** [anufrievroman/calcure#79]
Lately, I am unable to run it on Windows. It's installed but when I run `python -m calcure` it just does not start TUI interface and quits the process after a second without any errors. I was wondering if anyone recently was able to install it on Windows? Perhaps `windows-curses` library has changed or some other windows specific component?

**Comment by jose1711** [comment]
Have only tested with latest version from `pypi` (2.9.6). Running `pip install calcure` followed by `python -m calcure` returned error: `No module named: '_curses'`.

Had to run `pip install windows-curses`, then it worked.

OS: Windows 11 Pro, `python` 3.11.6 from MS Store

**Comment by anufrievroman** [comment]
Good to hear that, maybe it's just my problem. Previously it worked for me just like described in readme (like you described) but lately didn't work.

By the way, do you know if we can add windows-curses as a dependency to `setup.py` depending on the OS?

**Comment by jose1711** [comment]
> By the way, do you know if we can add windows-curses as a dependency to `setup.py` depending on the OS?

Sorry, I don't have much experience with packaging Python apps.

**Comment by anufrievroman** [comment]
Okay, since it seems to work for other people I'll close this.

**Comment by Bobrokus** [comment]
- terminal: wezterm
- shell: PowerShell 7
- OS: Windows

The same thing happens to me. When I run `python -m calcure` via my shell, it does nothing for a second and quits.

I was able to find some error code, though.

After running `python -m calcure` via `Win + R` error pops up.

![image](https://github.com/anufrievroman/calcure/assets/63786893/3e27e0c7-62af-43fc-b151-65c8bd874491)
*It's running in Windows Terminal now because it's set as the default terminal.*

However, it might not be an error from the code itself.

**Comment by csc530** [comment]
- terminal: windows terminal preview
- shell: PowerShell 7
- OS: Windows 11

I have the same problem. I found that with the scoop main/python (3.12.1) package, calcure wouldn't work but with the main/winpython (3.11.6.0) package, it does work as expected. I think it may have to do with some of the bundled packages with winpyhton.

I don't have much experience with Python to investigate if there's a missing dependency for the project for Windows, but I tried installing exclusively each package that comes with WinPython, but it still would close after a few seconds. So I don't think it's any *individual* package in winpython

**Comment by Bobrokus** [comment]
This [Stack Overflow answer](https://stackoverflow.com/a/51817869/20918903) might help

**Comment by anufrievroman** [comment]
Thank you everyone for reporting. I have a feeling that it's something about environments. Like someone on stack overflow points out, installation with pip might not work but installation with conda does. I mean, the package used to work in Windows and people did successfully installed the last version, so it's something about individual Python configuration.

At the moment, I don't have any windows installations at hand to check it. Could someone try installing it with conda or something like that?

**Comment by Bobrokus** [comment]
@anufrievroman What exactly is Conda? I haven't used it before.

**Comment by anufrievroman** [comment]
Me neither but it's like a python distribution, it comes from [Anaconda](https://www.anaconda.com), and there python is sort of isolated from the system.

**Comment by csc530** [comment]
I tried installing calcure with Conda/Anaconda's (scoop extras/anaconda3) *`pip install`*, *not* `conda install` and it too worked. Even though I'm fairly certain global pip installs isn't what it was designed for.

Conda also comes pre-packed with a variety of packages too

**Comment by Bobrokus** [comment]
### I tried to install calcure with `pipx`. It returned this:
![error](https://github.com/anufrievroman/calcure/assets/63786893/bfdcd7ea-c60f-4fa3-b6ae-e3deb01a9da3)
```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "D:\ctibo\.local\bin\calcure.exe\__main__.py", line 4, in <module>
  File "C:\Users\ctibo\AppData\Local\pipx\pipx\venvs\calcure\Lib\site-packages\calcure\__main__.py", line 5, in <module>
    import curses
  File "D:\Program Files\Scoop\apps\python\3.12.1\Lib\curses\__init__.py", line 13, in <module>
    from _curses import *
ModuleNotFoundError: No module named '_curses'
```

**Comment by anufrievroman** [comment]
@Bobrokus yes this error means that you need windows-curses library. So, when you install it, the error persists?

**Comment by Bobrokus** [comment]
@anufrievroman yes

**Comment by JH34tem** [comment]
Doesn't run on windows, there's no error message in cmd prompt but yea
